pymc3/theanof.py

```diff
@@ -1,10 +1,12 @@
 import numpy as np
+import theano
 
 from .vartypes import typefilter, continuous_types
 from theano import theano, scalar, tensor as tt
 from theano.gof.graph import inputs
-from theano.gof import Op
+from theano.gof import Op, Apply
 from theano.configparser import change_flags
+from theano.tensor.nlinalg import matrix_inverse
 from .memoize import memoize
 from .blocking import ArrayOrdering
 from .data import DataGenerator
```

Comment:
What exactly would you want to add to `__all__` ? Do you mean adding `LogDet` to it?